boa/boa-id-v1.jsonld

```diff
@@ -0,0 +1,35 @@
+{
+  "@context":
+  {
+    "@version": 1.1,
+    "bid": "https://sphereon-opensource.github.io/factom-vc-contexts/boa/boa-id-v1#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "schema": "http://schema.org/",
+
+    "BoaIdCredential" : "bid:BoaIdCredential",
+
+    "issueDate": {
+      "@id": "bid:issueDate",
+      "@type": "xsd:dateTime"
+    },
+    "validityDuration": {
```

Comment:
I know it was mentioned in the ticket, but are we sure a duration is what is asked? We either can have a duration or we can have the caller/API allow for durations, but simply have an end date. I am more inclined to use start and end dates then have a duration in the data model.

Reply:
Updated to `expirationDate`.

Reply:
Has since been changed into validFrom/To, so resolving